pkg/scheduler/metrics/metrics.go

```diff
@@ -27,7 +27,7 @@ const (
 	// SchedulerSubsystem - subsystem name used by scheduler
 	SchedulerSubsystem = "scheduler"
 	// SchedulingLatencyName - scheduler latency metric name
-	SchedulingLatencyName = "scheduling_latencies_summary"
+	SchedulingLatencyName = "scheduling_latency_seconds"
 
 	// OperationLabel - operation label name
 	OperationLabel = "operation"
```

Comment:
In the [naming](https://prometheus.io/docs/practices/naming/) section of the prometheus guide, they mention that,

> either the sum() or the avg() over all dimensions of a given metric should be meaningful.

When you define a label with values for "binding", "algorithm", and "e2e", then sum and avg are both meaningless.

For counters, it's clear to see how the "total" would be represented by the sum over each label value. I am not sure if histograms work the same way, but this smells odd to me.

Reply:
Ok. I think I understand your point.
I've removed the "e2e" label. This "e2e" is equivalent to the sum of "binding" and "algorithm".